Migrate backends/qualcomm to the new namespace

dbort opened the pull request:
Summary: Move the qualcomm backend out of the `torch::` namespace, and update to avoid using the `torch::` or `exec_aten::` namespaces.

Differential Revision: D64073805

shewu-quic reviewed (approved):
LGTM. Thanks for helping with this.

cccclai reviewed (approved):
Stamp on behalf of QCOMM team

dbort committed:
Migrate backends/qualcomm to the new namespace (#6025)

Summary:
Pull Request resolved: https://github.com/pytorch/executorch/pull/6025

Move the qualcomm backend out of the `torch::` namespace, and update to avoid using the `torch::` or `exec_aten::` namespaces.

Reviewed By: cccclai

Differential Revision: D64073805

chiwwang reviewed (approved):
I feel ok. @shewu-quic also feel ok then it's good to go from QC view!

dbort commented:
@pytorchbot cherry-pick --onto release/0.4 -c fixnewfeature

dbort commented:
@pytorchbot cherry-pick --onto release/0.4 -c fixnewfeature
